spec/features/admin/tutorials/admin_can_add_tutorials_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 describe "As an admin" do
-    scenario "can add a tutorial" do
```

Comment:
Let's add back a happy test for admin creating a tutorial with a video.

Reply:
We have decided to move forward with merging this branch and have added the happy path admin tutorial w/ video as an issue on our legacy code fixes.